hugo/content/_index.md

```diff
@@ -40,7 +40,7 @@ Few examples for a quick reference:
 
 * May be just close the JIRA? cause we just released a project and it is release review JIRA.
   ```groovy
-    def transitionInput = [ transition: [ name: 'Close'] ]
+    def transitionInput = [ transition: [ id: 31] ]
```

Comment:
This number can differ from instance to instance, can you make a comment about that this id is of transaction status and how to get it just in case?

Reply:
acknowledged